protocol/messenger.go

```diff
@@ -720,6 +720,20 @@ func (m *Messenger) Start() (*MessengerResponse, error) {
 		}
 	}
 
+	joinedCommunities, err := m.communitiesManager.Joined()
+	if err != nil {
+		return nil, err
+	}
+
+	for _, joinedCommunity := range joinedCommunities {
+		// resume importing message history archives in case
+		// imports have been interrupted previously
+		err := m.resumeHistoryArchivesImport(joinedCommunity.ID())
```

Comment:
If there's an error with a single community, you might end up some communities being processed and some not.
Is that problematic? Could the error handling instead of returning the error, log it and continue with the next community?
```

Reply:
Was considering that as well! And then I had a @cammellos in my head saying something like "generally we want to fail fast".

Happy to change that though if there's strong opinions! :D